src/lib/components/Segment.svelte

```diff
@@ -105,8 +105,6 @@
     border-radius: var(--border-radius);
     overflow: hidden;
 
-    width: 100%;
```

Comment:
Can you investigate why this was here an explain why it's no longer necessary?

Reply:
One part of the Segment restyling was to use whole width on mobile, instead of calculated width. And this rule was added by mistake because it's not necessary and lead to increased width because of the padding. 
`width: 100%;` with `box-sizing: border-box;` could also fix it, but better use less css then more when the result is the same.